README.md

```diff
@@ -27,29 +27,31 @@ Compile using babel:
 gulp babel
 ```
 
-Start a repl:
+Require Intercom:
 
 ```
```

Comment:
Sure, but I think it should be `bash` or `sh` here as its not javascript there, but bash command :)